### PR TITLE
Docs for two-step verification

### PR DESCRIPTION
**Pull request by snehaso**

### CODE REVIEW

File: source/docs/two-step-verification.md.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Two-step verification secures your account by requiring a one time password, in addition to your username and password, to access your account. This ensures that your account stays secured even if your password is compromised.
+
+
+### Turn on 2-Step Verification
+
+Go to your [account settings](https://semaphoreci.com/users/edit) page and then in the 2-Step Verification section, click on "Enable". On the next screen, you can choose the authenticator app for your phone and install it.
+
+<img src="/docs/assets/img/two-step-verification/enable.png" class="img-responsive">
+
+With an authenticator app, you scan a barcode code, and the app connects with the semaphore website to verify itself. It then displays a six-digit code, which you have to enter in a limited amount of time. When the time runs out, the app will generate a new code. Once you verify code successfully, two step verification is turned on for your account.
```
Comment (markoa) [inline source/docs/two-step-verification.md.erb]: semaphore => Semaphore

File: source/docs/two-step-verification.md.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### How to recover if you lose access to your phone
+
+Once you turn on two-step verification, Semphore generates the recovery codes for you. You can download them [here](https://semaphoreci.com/users/two_step_verification/recovery_codes). If you loose acess to your phone, you can still complete the second step of verification using a recovery code.
```
Comment (markoa) [inline source/docs/two-step-verification.md.erb]: - Semphore => Semaphore
- loose => lose

File: source/docs/two-step-verification.md.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+<img src="/docs/assets/img/two-step-verification/back-up-code-listed.png" class="img-responsive">
+
+To verify using recovery code, click "Use backup recovery codes" during two-step verification process. It will ask you to authenticate using the recovery code.
+
+<img src="/docs/assets/img/two-step-verification/authenticate-with-recovery-code.png" class="img-responsive">
+
+### Turn off 2-Step Verification
```
Comment (markoa) [inline source/docs/two-step-verification.md.erb]: Maybe change this title to "How to Turn Off 2-Step Verification", currently when I read it almost sounds like we recommend doing it. :)

**Comment by snehaso** [comment]
@markoa Thanks ! I've fixed the typos and have updated the PR.

File: source/docs/two-step-verification.md.erb [hunk 4]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
 category: Two-Step Verification
 ---
 
-Two-step verification secures your account by requiring a one time password, in addition to your username and password, to access your account. This ensures that your account stays secured even if your password is compromised.
+Two-step verification secures your account by requiring a one time password, in
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: one time => one-time

File: source/docs/two-step-verification.md.erb [hunk 5]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+addition to your username and password, to access your account. This ensures
+that your account stays secured even if your password is compromised.
 
 
 ### Turn on 2-Step Verification
 
-Go to your [account settings](https://semaphoreci.com/users/edit) page and then in the 2-Step Verification section, click on "Enable". On the next screen, you can choose the authenticator app for your phone and install it.
+Go to your [account settings](https://semaphoreci.com/users/edit) page and then
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: click
on "Enable" in the 2-Step Verification section.

File: source/docs/two-step-verification.md.erb [hunk 6]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+in the 2-Step Verification section, click on "Enable". On the next screen, you
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: next => following

File: source/docs/two-step-verification.md.erb [hunk 7]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+can choose the authenticator app for your phone and install it.
 
 <img src="/docs/assets/img/two-step-verification/enable.png" class="img-responsive">
 
-With an authenticator app, you scan a barcode code, and the app connects with the semaphore website to verify itself. It then displays a six-digit code, which you have to enter in a limited amount of time. When the time runs out, the app will generate a new code. Once you verify code successfully, two step verification is turned on for your account.
+With an authenticator app, you scan a barcode code, and the app connects with
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: barcode code => barcode

Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: connects => will connect

File: source/docs/two-step-verification.md.erb [hunk 8]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+the Semaphore website to verify itself. It then displays a six-digit code,
+which you have to enter in a limited amount of time. When the time runs out,
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: have to enter in a limited amount of time => you will need to enter within a limited time frame

File: source/docs/two-step-verification.md.erb [hunk 9]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+the app will generate a new code. Once you verify code successfully, two step
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: Once you verify code successfully, => Once you have successfully verified the code,

File: source/docs/two-step-verification.md.erb [hunk 10]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+verification is turned on for your account.
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: is turned on => will be turned on

File: source/docs/two-step-verification.md.erb [hunk 11]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
 
 ### How to recover if you lose access to your phone
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: How to recover => What to do

File: source/docs/two-step-verification.md.erb [hunk 12]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
 
-Once you turn on two-step verification, Semphore generates the recovery codes for you. You can download them [here](https://semaphoreci.com/users/two_step_verification/recovery_codes). If you loose acess to your phone, you can still complete the second step of verification using a recovery code.
+Once you turn on two-step verification, Semaphore generates the recovery codes
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: generates => will generate

File: source/docs/two-step-verification.md.erb [hunk 13]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
 <img src="/docs/assets/img/two-step-verification/back-up-code-listed.png" class="img-responsive">
 
-To verify using recovery code, click "Use backup recovery codes" during two-step verification process. It will ask you to authenticate using the recovery code.
+To verify using recovery code, click "Use backup recovery codes" during two-step
+verification process. It will ask you to authenticate using the recovery code.
 
 <img src="/docs/assets/img/two-step-verification/authenticate-with-recovery-code.png" class="img-responsive">
 
-### Turn off 2-Step Verification
+### How to Turn Off 2-Step Verification
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: How to Turn Off 2-Step Verification => How to turn off 2-step verification

File: source/docs/two-step-verification.md.erb [hunk 14]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
 
-Once you turn on two-step verification, Semphore generates the recovery codes for you. You can download them [here](https://semaphoreci.com/users/two_step_verification/recovery_codes). If you loose acess to your phone, you can still complete the second step of verification using a recovery code.
+Once you turn on two-step verification, Semaphore generates the recovery codes
+for you. You can download them [here](https://semaphoreci.com/users/two_step_verification/recovery_codes). If you lose acess to your phone,
+you can still complete the second step of verification using a recovery code.
 
 <img src="/docs/assets/img/two-step-verification/back-up-code-listed.png" class="img-responsive">
 
-To verify using recovery code, click "Use backup recovery codes" during two-step verification process. It will ask you to authenticate using the recovery code.
+To verify using recovery code, click "Use backup recovery codes" during two-step
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: during two-step => during the two-step

File: source/docs/two-step-verification.md.erb [hunk 15]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+verification process. It will ask you to authenticate using the recovery code.
 
 <img src="/docs/assets/img/two-step-verification/authenticate-with-recovery-code.png" class="img-responsive">
 
-### Turn off 2-Step Verification
+### How to Turn Off 2-Step Verification
 
-Go to your [account settings](https://semaphoreci.com/users/edit) and then in the 2-Step Verification section, click on "Disable". You will be asked to enter verification code generated through the authenticator app and then select "Disable 2-step verification".
+Go to your [account settings](https://semaphoreci.com/users/edit) and then in the
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: then in the
2-Step Verification section, click on "Disable".
=>
click on
"Disable" in the 2-Step Verification section.

File: source/docs/two-step-verification.md.erb [hunk 16]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
...
+2-Step Verification section, click on "Disable". You will be asked to enter
+verification code generated through the authenticator app and then select
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: verification code => the verification code

Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: app, and

File: source/docs/two-step-verification.md.erb [hunk 17]
```diff
@@ -4,30 +4,44 @@ title: Two Step Verification
 category: Two-Step Verification
 ---
 
-Two-step verification secures your account by requiring a one time password, in addition to your username and password, to access your account. This ensures that your account stays secured even if your password is compromised.
+Two-step verification secures your account by requiring a one time password, in
+addition to your username and password, to access your account. This ensures
+that your account stays secured even if your password is compromised.
 
 
 ### Turn on 2-Step Verification
```
Comment (dunjardl) [inline source/docs/two-step-verification.md.erb]: Turn on 2-Step Verification => Turn on 2-step verification

**Comment by snehaso** [comment]
@dunjardl I've updated the text. Thanks for the help. The doc looks far better than before and simple to follow. I appreciate your help. :smile:

**Pull request merged**